lib/producer.mjs

```diff
@@ -23,7 +23,7 @@ export const start = async ({ clientId, messenger }) => {
       console.log(`${clientId} / Producer.Ready`);
 
       for await (const { topic, message, options } of messenger()) {
-        dispatcher.dispatch(topic, message, options);
+        dispatcher.dispatch(topic, Buffer(JSON.stringify(message)), options);
```

Comment:
By design, we wanted to just dispatch the message as is.
The string was supposed to be buffered in the message producer, as listed here:

Could you elaborate a bit more on why you would want to change this behaviour?


<img width="874" alt="image" src="https://user-images.githubusercontent.com/5674833/175515341-93cc9bb0-d442-4e7d-a9c6-84dda4f3c8f2.png">